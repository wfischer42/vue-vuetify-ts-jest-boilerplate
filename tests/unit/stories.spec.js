import registerRequireContextHook from 'babel-plugin-require-context-hook/register'
registerRequireContextHook()

import Vue from 'vue'
Vue.config.ignoredElements = [
  'v-app',
  // 'another-web-component',
  // // Use a `RegExp` to ignore all elements that start with "ion-"
  // // 2.5+ only
  // /^v-/,
]

import initStoryshots from '@storybook/addon-storyshots'
initStoryshots({ configPath: './config/storybook' })
