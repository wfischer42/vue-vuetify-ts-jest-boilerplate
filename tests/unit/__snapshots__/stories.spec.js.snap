// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Hello World Default View 1`] = `
<div
  class="application theme--light"
  data-app="true"
  id="app"
>
  <div
    class="application--wrap"
  >
    <main
      class="v-content"
      style="padding: 0px 0px 0px 0px;"
    >
      <div
        class="v-content__wrap"
      >
        <div
          class="container"
        >
          <div
            class="layout text-xs-center wrap"
          >
            <div
              class="flex xs12"
            >
              <div
                class="v-responsive v-image my-3"
                style="height: 200px;"
              >
                <div
                  class="v-image__image v-image__image--preload v-image__image--contain"
                  style="background-position: center center;"
                />
                <div
                  class="v-responsive__content"
                />
              </div>
            </div>
             
            <div
              class="flex mb-4"
            >
              <h1
                class="display-2 font-weight-bold mb-3"
              >
                Welcome to Vuetify
              </h1>
               
              <p
                class="subheading font-weight-regular"
              >
                
        For help and collaboration with other Vuetify developers,
        
                <br />
                please join our online
        
                <a
                  href="https://community.vuetifyjs.com"
                  target="_blank"
                >
                  Discord Community
                </a>
              </p>
            </div>
             
            <div
              class="flex mb-5 xs12"
            >
              <h2
                class="headline font-weight-bold mb-3"
              >
                What's next?
              </h2>
               
              <div
                class="layout justify-center"
              >
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com/components/api-explorer"
                  target="_blank"
                >
                  Explore components
                </a>
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com/layout/pre-defined"
                  target="_blank"
                >
                  Select a layout
                </a>
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com/getting-started/frequently-asked-questions"
                  target="_blank"
                >
                  Frequently Asked Questions
                </a>
              </div>
            </div>
             
            <div
              class="flex xs12 mb-5"
            >
              <h2
                class="headline font-weight-bold mb-3"
              >
                Important Links
              </h2>
               
              <div
                class="layout justify-center"
              >
                <a
                  class="subheading mx-3"
                  href="https://vuetifyjs.com"
                  target="_blank"
                >
                  Documentation
                </a>
                <a
                  class="subheading mx-3"
                  href="https://community.vuetifyjs.com"
                  target="_blank"
                >
                  Chat
                </a>
                <a
                  class="subheading mx-3"
                  href="https://madewithvuetifyjs.com"
                  target="_blank"
                >
                  Made with Vuetify
                </a>
                <a
                  class="subheading mx-3"
                  href="https://twitter.com/vuetifyjs"
                  target="_blank"
                >
                  Twitter
                </a>
                <a
                  class="subheading mx-3"
                  href="https://medium.com/vuetify"
                  target="_blank"
                >
                  Articles
                </a>
              </div>
            </div>
             
            <div
              class="flex xs12 mb-5"
            >
              <h2
                class="headline font-weight-bold mb-3"
              >
                Ecosystem
              </h2>
               
              <div
                class="layout justify-center"
              >
                <a
                  class="subheading mx-3"
                  href="https://github.com/vuetifyjs/vuetify-loader"
                  target="_blank"
                >
                  vuetify-loader
                </a>
                <a
                  class="subheading mx-3"
                  href="https://github.com/vuetifyjs/vuetify"
                  target="_blank"
                >
                  github
                </a>
                <a
                  class="subheading mx-3"
                  href="https://github.com/vuetifyjs/awesome-vuetify"
                  target="_blank"
                >
                  awesome-vuetify
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
  </div>
</div>
`;

exports[`Storyshots MyButton with JSX 1`] = `
<div
  class="application theme--light"
  data-app="true"
  id="app"
>
  <div
    class="application--wrap"
  >
    <main
      class="v-content"
      style="padding: 0px 0px 0px 0px;"
    >
      <div
        class="v-content__wrap"
      >
        <button
          class="v-btn theme--light"
          type="button"
        >
          <div
            class="v-btn__content"
          >
            With JSX
          </div>
        </button>
      </div>
    </main>
  </div>
</div>
`;

exports[`Storyshots MyButton with some emoji 1`] = `
<div
  class="application theme--light"
  data-app="true"
  id="app"
>
  <div
    class="application--wrap"
  >
    <main
      class="v-content"
      style="padding: 0px 0px 0px 0px;"
    >
      <div
        class="v-content__wrap"
      >
        <button
          class="v-btn theme--light"
          type="button"
        >
          <div
            class="v-btn__content"
          >
            üòÄ üòé üëç üíØ
          </div>
        </button>
      </div>
    </main>
  </div>
</div>
`;

exports[`Storyshots MyButton with text 1`] = `
<div
  class="application theme--light"
  data-app="true"
  id="app"
>
  <div
    class="application--wrap"
  >
    <main
      class="v-content"
      style="padding: 0px 0px 0px 0px;"
    >
      <div
        class="v-content__wrap"
      >
        <button
          class="v-btn v-btn--disabled theme--light"
          disabled="disabled"
          type="button"
        >
          <div
            class="v-btn__content"
          >
            Hello Storybook
          </div>
        </button>
      </div>
    </main>
  </div>
</div>
`;
